app/assets/stylesheets/modules/style.scss

```diff
@@ -0,0 +1,88 @@
+@import "variables";
+
+body {
+  background-image: image-url("default.jpg");
+  background-repeat: no-repeat;
+  background-attachment:fixed;
+  background-size: cover;
+  text-align: center;
+  color: $white;
+  height: 100%;
+  width: 100%;
+  font-family: Lato;
+  position: absolute;
+}
+
+.container-wrapper {
+  background-color: rgba( $black, .4 );
+  text-align: center;
+  min-height: 100%;
+  height: auto;
+  width: 100%;
+}
+
+.container {
+  /* background-color: rgba( $black, .6 ); */
+  top: 20px;
+  border-radius: 25px;
+  padding: 1%, 5%, 5%, 5%;
+  text-align: center;
+  display: inline-block;
+  position: relative
+}
+
+.right {
+  text-align: right;
+}
+
+.center {
+  margin-left:auto;
+  margin-right:auto;
+}
+
+h1 {
+  font-family: Rock Salt;
+  text-align: center;
+}
+
+h2 {
+  font-family: Rock Salt;
+  font-weight: bold;
+}
+
+.image-attribution {
+  position: fixed;
+  bottom: 0px;
+  right: 0px;
+  font-size: 10px;
+  padding: 2px;
+  color: $white;
+}
+
+table {
+  color: $white;
+  margin: 8px;
+  border-color: $red;
+  border-width: 1px;
+  border-radius: 8px;
+}
+
+img {
+  height: auto;
+  width: auto;
+}
+
+a:link, a:visited, a:active {
+  text-decoration: none;
+  color: $white;
+  text-decoration:none;
```

Comment:
Property `text-decoration` already defined on line 76<br>Colon after property should be followed by one space